src/ui/libs/DatalensChartkit/modules/axios/axios.ts

```diff
@@ -55,11 +58,22 @@ export function initConcurrencyManager(maxConcurrentRequests: number) {
 
 client.interceptors.response.use(
     (data) => data,
-    (error) => {
+    async (error) => {
         if (error?.response?.status === 451) {
             showReadOnlyToast();
         }
 
+        if (error && error.response && error.response.status === 498) {
```

Comment:
I suggest you to use `axios.isAxiosError(error)` here

Reply:
agree, added